powershell-yaml.psm1

```diff
@@ -87,7 +87,11 @@ function Convert-ValueToProperType {
                 return $datetime
             }
         }
-            
+
+        if ($Node.Style -eq 'Plain' -and $Node.Value -in '','~','null','Null','NULL') {
```

Comment:
See previous comment. I think we can leave this out. YamlDotNet seems to do the right thing for ```$null``` and and empty string should be deserialized as empty string.

For example:

```powershell
PS /tmp/aaa/powershell-yaml> $aString = @{"iAmAString"=""}    
PS /tmp/aaa/powershell-yaml> $aString["iAmAString"].GetType().FullName
System.String
PS /tmp/aaa/powershell-yaml> $asYaml = ConvertTo-Yaml $aString
PS /tmp/aaa/powershell-yaml> $asYaml
iAmAString: 

PS /tmp/aaa/powershell-yaml> $backToHash = ConvertFrom-Yaml $asYaml
PS /tmp/aaa/powershell-yaml> $backToHash["iAmAString"].GetType().FullName
System.String
PS /tmp/aaa/powershell-yaml> $backToHash["iAmAString"].GetType().FullName -eq $aString["iAmAString"].GetType().FullName
True
```

is desirable.

Reply:
The new test case shows why it's necessary.  It fails if this is removed. :)